hudi-integ-test/pom.xml

```diff
@@ -117,7 +117,7 @@
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
-      <version>20.0</version>
+      <version>15.0</version>
```

Comment:
why the downgrade? this should be fine for the integ test , if its passing consistently right